test/benchmarking/test_stash_circular_references.py

```diff
@@ -0,0 +1,26 @@
+# pylint: disable=missing-class-docstring,missing-function-docstring,protected-access
+
+import pytest
+from termcolor import cprint
+
+from dsp_tools.utils.xml_utils import parse_and_clean_xml_file
+from dsp_tools.utils.xmlupload.stash_circular_references import remove_circular_references
+from dsp_tools.utils.xmlupload.xmlupload import _extract_resources_from_xml
+
+
+def test_get_length_ok_resources() -> None:
+    test_root = parse_and_clean_xml_file("testdata/generate_test_data/test_circular_references_1.xml")
+    resources = _extract_resources_from_xml(test_root, "simcir")
+    ok_resources, _, _ = remove_circular_references(resources, False)
+    print_str = (
+        f"\n\n---------------------\n"
+        f"Original length: 26\n"
+        f"New length: {len(ok_resources)}"
+        f"\n---------------------\n"
+    )
+    cprint(text=print_str, color="yellow", attrs=["bold"])
```

Comment:
yellow is perfect for dark mode, but I don't know if it is good on white background. Red would fit both backgrounds

Reply:
I didn't want to use red because the failed test is red so I wanted an unused colour. But I could do blue...?

Reply:
I tried it out with the black background, but it looks very strange. I also tried it out in light mode and the yellow is still very much readable.